shaders/src/shading_parameters.fs

```diff
@@ -26,7 +26,7 @@ void computeShadingParams() {
 #endif
 
     shading_position = vertex_worldPosition;
-    shading_view = normalize(frameUniforms.cameraPosition - shading_position);
+    shading_view = -normalize(shading_position);
```

Comment:
Add a comment explaining why the view vector is computed this way, it can be surprising.

Reply:
Done